neo/Helper.cs

```diff
@@ -307,5 +308,20 @@ internal static long WeightedAverage<T>(this IEnumerable<T> source, Func<T, long
                 yield return resultSelector(item, weight);
             }
         }
+
+
+        /// <summary>
+        /// load configuration with different Environment Variable
+        /// </summary>
+        /// <param name="config"></param>
+        /// <returns></returns>
+        public static IConfigurationRoot LoadConfig(string config)
+        {
+            var fileEnv = Environment.GetEnvironmentVariable($"NEO_{config.ToUpperInvariant()}_FILE");
```

Comment:
If we read from `NEO_XXX_FILE`, this method is not reusable. We can move it from `Helper.cs` to `ProtocolSettings.cs`. And mark it as `private`.

Reply:
I think this method can be used for all the "config" file loading(e.g. protocal.json, config.json, all other plugin.json). Batch change the config files by just setting one environment variable value.

Reply:
Then we should have only one environment variable for switching network.